scripts/alcf_mira_cmake

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+EXTRA_ARGS=$@
+BLAS_DIR=/soft/libraries/alcf/20130312/gcc/BLAS
+LAPACK_DIR=/soft/libraries/alcf/20130312/gcc/LAPACK
+BOOST_DIR=/soft/libraries/boost/1.58.0/cnk-gcc/V1R2M2-20150523
+TRILINOS_DIR=/home/lebrung/projects/dtk/source/trilinos/12.8.1
+ARGS=(
+    -D CMAKE_BUILD_TYPE=Debug
```

Comment:
Sorry for being pushy. What do you need help with? As far as I can see the script itself looks fine.